### PR TITLE
Publish notebook from container.

### PR DESCRIPTION
**Pull request by DailyDreaming**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,10 +20,14 @@ mypy:
...
 notebooks:=$(wildcard notebooks/*.py)
-$(notebooks): lint mypy
-	python $@
-	callysto $@ > $(@:.py=.ipynb)
-	scripts/publish.sh $@ $(@:.py=.ipynb)
+PIP_REQS:=$(shell cat $(LOCAL_ROOT_DIR)/requirements.txt | tr "\n" " ")
+$(notebooks): clean lint mypy
+	docker exec $(CONTAINER) bash -c "$(LEO_PIP) install --upgrade $(PIP_REQS)"
+	docker exec $(CONTAINER) mkdir -p notebooks
+	docker cp $(LOCAL_ROOT_DIR)/$@ $(CONTAINER):$(CONTAINER_ROOT_DIR)/$@
```
Comment (xbrianh) [inline Makefile:27]: The entire repo directory is mounted into the container under `CONTAINER_REPO_ROOT="/$(basename ${BDCAT_NOTEBOOKS_HOME})"`. I think this line can be removed?

Comment (DailyDreaming) [reply Makefile]: Should be addressed.

File: Makefile [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+CONTAINER_HOME_DIR=/home/jupyter-user
+CONTAINER_REPO_DIR=$(CONTAINER_HOME_DIR)/$(shell basename ${LOCAL_ROOT_DIR})
+LEO_PYTHON=$(CONTAINER_HOME_DIR)/miniconda/bin/python
+LEO_PIP=$(CONTAINER_HOME_DIR)/miniconda/bin/pip
+CALLYSTO=callysto
 
 all: test
 
 test: lint mypy
+	@echo $(CALLYSTO)
```
Comment (xbrianh) [inline Makefile:15]: We can probably remove this line.

**Review by xbrianh — APPROVED** [review]
See comment, but LGTM

File: Makefile (hunk 2 above)
Comment (DailyDreaming) [reply Makefile]: D'oh.  Forgot this one.  Thanks!

**Pull request merged**